HAL_extern.h

```diff
@@ -153,7 +152,6 @@ void    initConfKey(void) {
 
 #ifndef USE_OWN_ISR_PCINT2_vect
 	ISR (PCINT2_vect) {
-		pcInt[2].prev = pcInt[2].cur;
```

Comment:
Warum fällt pcInt[(0-2)].prev weg?
Das fällt dann spätestens in chkPCINT() auseinander. Denn dort wird das noch verwendet.